src/main/java/com/google/devtools/build/lib/runtime/BlazeOptionHandler.java

```diff
@@ -74,6 +78,14 @@ public final class BlazeOptionHandler {
           "client_env",
           "client_cwd");
 
+  // All options set on this pseudo command are inherited by all commands, with unrecognized options
+  // resulting in an error.
+  private static final String ALL_OPTIONS_PSEUDO_COMMAND = "always";
```

Comment:
I would name the constants according to the pseudo command name to avoid introducing more vocabulary.